src/test/java/org/openmrs/contrib/qaframework/page/AttachmentsPage.java

```diff
@@ -0,0 +1,55 @@
+package org.openmrs.contrib.qaframework.page;
+
+
+import org.openmrs.reference.page.ClinicianFacingPatientDashboardPage;
+import org.openmrs.uitestframework.page.Page;
+import org.openqa.selenium.By;
+
+public class AttachmentsPage extends Page{
+	
+	private static final By ATTACHMENTS_LINK = By.className("icon-share-alt edit-action right");
+	private static final By ATTACH_FILE = By.className("dz-default dz-message ng-binding");
+	private static final By ADD_CAPTION = By.className("ng-pristine ng-valid ng-empty ng-touched");
+	private static final By UPLOAD_FILE = By.className("confirm ng-binding");
+	private static final By CLEAR_FORMS = By.className("ng-binding");
+	public static final By ICON_CAMERA = By.className("icon-camera");
+	public static final By SAVE_BUTTON = By.className("icon-ok");
+	
+	public AttachmentsPage(ClinicianFacingPatientDashboardPage clinicianFacingPatientDashboardPage) {
+		super(clinicianFacingPatientDashboardPage);
+	}
+
+	@Override
+	public String getPageUrl() {
+		return "/attachments/attachments.page";
+	}
+
+	public void clickOnAttachmentAppLink() {
+		driver.findElement(ATTACHMENTS_LINK).click();
```

Comment:
move this into the dashboard page within distro

Reply:
> move this into the dashboard page within distro

 I understand referencing this element in my definition steps  is accessed through a library after moving it to the distro. Do I need to raise a PR to the `distro repo` since this feature does not exist in the upstream master. 
cc: @kaweesi